runtime-libraries/maven-repository-zip/pom.xml

```diff
@@ -999,6 +999,10 @@
       <groupId>org.kie</groupId>
       <artifactId>kie-dmn-validation</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-dmn-test-resources</artifactId>
+    </dependency>
```

Comment:
I think it might require the classifier

maybe missing also the sources

Reply:
@porcelli yup - trying locally and fixing.
No sources needed, though

Reply:
From local test it seems it is not required, andiway just add the `classifier`